### DISCUSSION TITLE
ACF DAMAGE CHANGE IDEA

### DISCUSSION THREAD

**Discussion by d4v3cc** [TAServers/issues#30] [Ideas]
sooooo recently faisal got amazing idea of making the acf damage same as the pvp/build state
so that means if ur in pvp, your props can be damaged, if ur in build they cant. SHRIMPLE AS THAT!!

And why the hell not?!?!??!

**Comment by kingfaisalI** [comment]
has been implemented in a couple servers, so why not in TAS?

**Comment by Derpius** [comment]
Do you have links to the relevant functions etc to control acf damage on a _per-player_ basis?

**Reply by d4v3cc** [reply]
@kingfaisalI

**Reply by kingfaisalI** [reply]
xander already linked one

**Comment by Xandertron** [comment]
https://github.com/ACF-Team/ACF-3/blob/afefc2a0f2ebe7be8db3ee4b9460cc3cd4233bfa/lua/acf/damage/permissions/permissions_sv.lua
